### PR TITLE
Fixed mangled Tier data & rendering correctly by bundleId in url and re-rendering on change.

### PR DESCRIPTION
**Pull request by theofficetroll**
Fixed mangled Tier data.
Commented out Item rendering until it is fixed.
Changed so Tier component was re-rendering on prop change.
Updated so BundleId was pulling from url.

https://trello.com/c/WTkany0a

### CODE REVIEW

File: server/index.js [hunk 1]
```diff
@@ -57,8 +68,16 @@ app.get('/itemsIncluded/:tierId', (req, res) => {
...
+  .then(data => {
+    let itemsData = {
+      item1: data[0].dataValues.itemId,
+      item2: data[1].dataValues.itemId,
+      item3: data[2].dataValues.itemId,
+      item4: data[3].dataValues.itemId,
+      item5: data[4].dataValues.itemId,
+      item6: data[5].dataValues.itemId,
+    }
+    res.json(itemsData);
```
Comment (kraithe) [inline server/index.js:80]: Looks good here and above.

File: src/components/item.jsx [hunk 2]
```diff
@@ -51,11 +51,11 @@ class Item extends React.Component {
     return (
       <div className="item-view">
         ItemId: {this.state.itemId} <br/>
-        Name: {this.state.itemName} <br/>
+        {/* Name: {this.state.itemName} <br/>
         Systems: {this.state.availability} <br/>
         Good Reviews: {this.state.feedback.positiveReviews}% <br/>
         Total Reviews: {this.state.feedback.numberOfReviews} <br/>
-        Reviews From: {this.state.feedback.reviewsFrom} <br/>
+        Reviews From: {this.state.feedback.reviewsFrom} <br/> */}
```
Comment (kraithe) [inline src/components/item.jsx:58]: Looks like you're commenting out (down to a primary key) for debugging, which is fine.

File: src/components/tier.jsx [hunk 3]
```diff
@@ -61,9 +64,9 @@ class Tier extends React.Component {
 
     return (
       <div>
-        <div className="tier">
+        <div key="this.props.tierId" className="tier">
```
Comment (kraithe) [inline src/components/tier.jsx:67]: I think what you want is `${this.props.tierId}`.
Double quotes without curly brackets probably won't give you what you're looking for

File: src/index.jsx [hunk 4]
```diff
@@ -21,16 +21,18 @@ class Tiers extends React.Component {
   }
 
   getBundleData() {
-    let bundleId = this.state.bundleId;
+    var url = new URL(window.location);
+    var bundleId = url.searchParams.get('bundle');
```
Comment (kraithe) [inline src/index.jsx:25]: Nice refactor!

File: src/components/tier.jsx (hunk 3 above)
Comment (theofficetroll) [reply src/components/tier.jsx:67]: Oops, thank you.

**Review by kraithe — COMMENTED** [review]
Vast majority of everything looks good and I take it the refactors fit your intended data flow.
Noted one missing set of { curly brackets } in a JSX tag

**Pull request merged**